lightkurve/lightcurve.py

```diff
@@ -1070,6 +1072,65 @@ def estimate_cdpp(self, transit_duration=13, savgol_window=101,
         cdpp_ppm = np.std(mean) * 1e6
         return cdpp_ppm
 
+    def query_solar_system_objects(self, cadence_mask='outliers', radius=None, sigma=3, cache=True):
+        '''Query to find what solar system are in a lightcurve or targetpixel file.
+
+        This function relies on The Virtual Observatory Sky Body Tracker (skybot) service which can be found here:
+            http://vo.imcce.fr/webservices/skybot/
+
+        Parameters
+        ----------
+        cadence_mask : str or bool
+            mask in time to select which frames or points should be searched for SSOs.
+            Default "outliers" will search for SSOs at points that are `sigma` from the mean.
+            "all" will search all cadences. Pass a boolean array with values of "True"
+            for times to search for SSOs.
+        radius : optional, float
+            Radius to search for bodies. If None, will set a sensible value.
+        sigma : optional, float
+            If `cadence_mask` is set to `"outlier"`, `sigma` will be used to identify
+            outliers.
+
+        Returns
+        -------
+        result : pandas.DataFrame
+            DataFrame containing the list objects in frames that were identified to contain
+            SSOs.
+        '''
+
+        location = self.mission.lower()
+
+        if isinstance(cadence_mask, str):
+            if cadence_mask == 'outliers':
+                cadence_mask = self.remove_outliers(sigma=sigma, return_mask=True)[1]
+
+        if cadence_mask == 'all':
+            cadence_mask = np.ones(len(self.time)).astype(bool)
+
+        elif not isinstance(cadence_mask, np.ndarray):
+            raise ValueError('Pass a cadence_mask method or a cadence_mask')
+
+        if (location == 'kepler') | (location == 'k2'):
+            pixel_scale = 4
+        if location == 'tess':
+            pixel_scale = 27
```

Comment:
Hard-coding the pixel scale isn't very generic (e.g. what about TPFs created using the factory).

I suggest adding a `tpf.pixel_scale` property, which could `return tpf.hdu[1].header['11PC5'] * 3600.`.

Reply:
This seems like a larger fix, because we would need to then think about how this propagates to lightcurves. Pixel scale is also hard coded in the interact functions, can I add this as an issue to fix for our next version?